server/src/helpers/translation.js

```diff
@@ -47,12 +49,38 @@ const getTranslatedText = async (text, originalLanguage) => {
   }
 }
 
+const readTranslationMockFile = async file => {
+  try {
+    const rawData = await fs.readFile(file)
+    const data = JSON.parse(rawData)
+    return data
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error(error)
```

Comment:
Mieux vaut préciser le contexte de l'erreur, ça permettra de mieux comprendre quand si on la voit passer dans les logs
```suggestion
    console.error('Error while reading the translation mock file', file, error)
```